src/components/shared/svg/EmailListArrow/EmailListArrow.jsx

```diff
@@ -0,0 +1,17 @@
+import styles from './EmailListArrow.module.scss';
```

Comment:
Is this component an arrow? If so I would rename it so that we can re-use it elsewhere. "EmailListArrow" is specific, if its just "Arrow" or "RightArrrow" you can use it again elsewhere should we need it.